packages/api/database/factory.js

```diff
@@ -49,8 +49,6 @@ Factory.blueprint('App/Models/User', async (faker, i, data) => {
 		address: faker.string(),
 		address2: faker.string(),
 		district: faker.string(),
-		city: faker.string(),
-		state: faker.string(),
```

Comment:
You should add new fields here instead of removing old ones

Reply:
In factory, we often don't include fields with foreign key. If I desire create a new user with a city i can make:
`const userFactory = await Factory.model('App/Models/user').create({city_id:1111});`